consensus/phases/communicator.go

```diff
@@ -257,6 +260,7 @@ func (nc *NaiveCommunicator) ExchangePhase1(
 	for {
 		select {
 		case res := <-nc.phase1result:
+			metrics.ConsensusPacketsRecv.WithLabelValues("phase 1").Inc()
```

Comment:
I don't think that we should count consensus packets here because they can be filtered later. It is better to record all the packets after `ExchangePhaseN` in the phase code (where we have log record `received packets: %d/%d`)

Reply:
done